### ISSUE TITLE
warn if there is a division by zero happening

### ISSUE THREAD

**Issue by matthiaskrgr** [openvalidation/openvalidation#22]
rules
````
1 AS ONE

0 AS ZERO

ONE / ZERO  AS TWO

your age must be TWO
````

schema: ` age: 0`

There is no warning that we divide by zero.
It seems that when this happen, the validation will never suceed no matter what value I enter in the playground.

We should warn somehow that there was a division by zero.

**Comment by jages** [comment]
Agreed, we do not want to, however, check each nested call for zero.
I'm taking somewhat of a 'trust the programmer' approach to this - @lionelpa any thoughts?
